test/microbench/org/apache/cassandra/test/microbench/ZeroCopyStreamingBenchmark.java

```diff
@@ -80,8 +80,8 @@
 import org.openjdk.jmh.annotations.Warmup;
 
 /**
- * Please ensure that this benchmark is run with entire_sstable_stream_throughput_outbound_megabits_per_sec
- * set to a really high value otherwise, throttling will kick in and the results will not be meaningful.
+ * Please ensure that this benchmark is run with stream_throughput_outbound set to a
```

Comment:
1.) I'm noticing that `entire_sstable_stream_throughput_outbound_megabits_per_sec` still seems to exist in the YAML and `Config`. Did we want to update that to `entire_sstable_stream_throughput_outbound` and give it units?

2.) Should this comment mention `entire_sstable_stream_throughput_outbound` (assuming #1 above) instead?

Reply:
Yes, it is a topic for me after the next rebase, thanks. I think there are also a few more recent parameters to be added but let's first deal with the framework and I will rebase and add the newest ones too. Thanks